packages/ui/src/helpers/authenticator/utils.ts

```diff
@@ -109,13 +123,11 @@ export const defaultAuthHubHandler: HubHandler = async (data, service) => {
  */
 export const listenToAuthHub = (
   service: AuthInterpreter,
-  handler: HubHandler = defaultAuthHubHandler
+  handler: AuthMachineHubHandler = defaultAuthHubHandler
```

Comment:
Renamed `HubHandler` -> `AuthMachineHubHandler` because this is just a wrapped function inside `HubHandler`. Any naming nits are welcome.